src/browser/app/App.react.js

```diff
@@ -17,14 +17,20 @@ import {connect} from 'react-redux';
 export default class App extends Component {
 
   static propTypes = {
+    actions: PropTypes.object.isRequired,
     children: PropTypes.object.isRequired,
     location: PropTypes.object.isRequired,
     msg: PropTypes.object.isRequired,
-    users: PropTypes.object.isRequired
+    persistence: PropTypes.object.isRequired
```

Comment:
I like the idea that you have some persistence store on server/browser and native, which has same API and you are not accessing it by direct access but through specified API in persistence. Then you are able to have persistence in appState so you can easily access all data from persistence in components too.